frontend/src/pages/meditation/components/meditation-sidebar/meditation-sidebar.tsx

```diff
@@ -0,0 +1,110 @@
+import meditationPlaceholder from '#assets/img/meditation-image-placeholder.png';
+import {
+  Button,
+  Card,
+  Search,
+  Sidebar,
+  SidebarBody,
+  SidebarHeader,
+} from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import {
+  useAppDispatch,
+  useCallback,
+  useRef,
+  useSearch,
+} from '#libs/hooks/hooks.js';
+import { actions as meditationActions } from '#slices/meditation/meditation.js';
+
+import { MeditationModal } from '../meditation-modal/meditation-modal.js';
+import styles from './styles.module.scss';
+
+const mockedMeditations = [{ id: 1, name: 'Meditation' }];
+
+const mockedSelectedMeditation = {
+  id: 1,
+};
```

Comment:
it's not a mocked data, we won't receive it from backend
```suggestion
const navigationItems = [{ name: 'Meditation', path: AppRoute.MEDITATION }];
```

Reply:
Done